design-guidelines/copy-and-content/date-time-formatting.md

```diff
@@ -0,0 +1,162 @@
+
+### Time of day
+
+Times are displayed using the 24-hour clock, and a leading 0 is used for single-digit hours. For example, we would display "13:00", not "01:00 PM".
+
+#### Time Zones
+
+Time zones should not be displayed because we should opt for showing the local time. The exception to this rule is when we need to display a timestamp using the UTC time standard. When displaying a UTC timestamp, please follow the W3C formatting guidelines: [W3C Date and Time Formats](https://www.w3.org/TR/NOTE-datetime#:~:text=Times%20are%20expressed%20in%20UTC,mm%22%20minutes%20ahead%20of%20UTC.)
+
+### Time Ranges
+
+The values in a time range should be separated by an en dash with 1 space on either side. For example, "12:00 – 18:00" or "6 May – 10 May".
+
+In the rare case a time range is being used in a sentence, the en dash should not be used. For example, "The cluster was active from 12:00 until 18:00".
+
+When displaying a time range, there should be some indication for the duration of the time range. The context in which the time range is being display should inform how precise of a duration should be shown. For example, given the range "12:00:00 – 18:24:30", we could display the duration as "6 hours", "6 hours and 24 minutes", or "6 hours, 24 minutes and 30 seconds" depending on how much detail the user needs to know about the time range.
+
+Both timestamps in a time range should be formatted the same way. For example, "01 Jan 2020 – 02 Feb 2020" is correct, but "01 Jan 2020 14:00 – 02 February" is incorrect.
+
+### Granularity
+
+Depending on the context of the absolute timestamp, there are different levels of detail a time can be shown. If the year of a date is not important, and the date is within the current year, the year may be excluded from the timestamp. The examples below are using full month names and full weekday names, but the abbreviated names are also acceptable.
+
+| Granularity         | Value (w/ year)               | Value (w/o year)         |
+| ------------------- | ----------------------------- | ------------------------ |
+| Date                | 01 January 2020               | 01 January               |
+| Date and weekday    | Monday, 01 January 2020       | Monday, 01 January       |
+| Date and time       | 01 January 2020 14:00         | 01 January 14:00         |
+| Date, time, weekday | Monday, 01 January 2020 14:00 | Monday, 01 January 14:00 |
+
+## Relative time
+
+Relative times may be used when users just need to know the approximate time of an event. Relative time formatting can feel more conversational than using an absolute timestamp.
+
+### Past
+
+| When       | Value                      |
+| ---------- | -------------------------- |
+| < 1 minute | just now                   |
+| < 1 hour   | {n} minutes ago            |
+| < 24 hours | {n} hours ago              |
+| < 7 days   | {n} days ago               |
```

Comment:
Hey Mike, should we consider a situation like this?
when time =24 hours, the value is '{n} day ago'
when time =1 month, the value is '{n} month ago'
...

Reply:
Sure, I will be more specific about not pluralizing when there is just 1 day/month/etc